compiler/ast/zed/value.go

```diff
@@ -29,15 +34,26 @@ type Any interface {
 	anyNode()
 }
 
+// Only Primitive and TypeValue are used in zed parser so only these are given
+// ast.Node features.
+
+var (
+	_ Node = (*Primitive)(nil)
+	_ Node = (*TypeValue)(nil)
+)
+
```

Comment:
No need for this since we'll get errors in compiler/semantic/expr.go and zfmt/ast.go if these types don't implement this interface.
```suggestion
```